modules/apps/captcha/captcha-rest-impl/rest-openapi.yaml

```diff
@@ -0,0 +1,119 @@
+components:
+    schemas:
+        Captcha:
+            properties:
+                captchaToken:
+                    type: string
+                image:
+                    type: string
+        FormCaptcha:
+            properties:
+                answer:
+                    type: string
+                captchaToken:
+                    type: string
+            required:
+                - captchaToken
+        GenericError:
+            description:
+                https://www.schema.org/GenericError
+            properties:
+                message:
+                    type: string
+            type: object
+info:
+    description:
+        "CaptchaRest REST API. A Java client JAR is available for use with the group ID 'com.liferay', artifact ID
+        'com.liferay.captcha.rest.client', and version '1.0.0'."
+    license:
+        name: "Apache 2.0"
+        url: "http://www.apache.org/licenses/LICENSE-2.0.html"
+    title: "Liferay Captcha REST"
+    version: v1.0
+openapi: 3.0.1
+paths:
+    /simple:
+        get:
+            operationId: getSimpleCaptcha
+            responses:
+                200:
+                    content:
+                        application/json:
+                            schema:
+                                $ref: "#/components/schemas/Captcha"
+                        application/xml:
+                            schema:
+                                $ref: "#/components/schemas/Captcha"
+                    description:
+                        Send a captcha token and image.
+                500:
+                    content:
+                        application/json:
+                            schema:
+                                $ref: "#/components/schemas/FormCaptcha"
+                        application/xml:
+                            schema:
+                                $ref: "#/components/schemas/FormCaptcha"
+                    description:
+                        Unexpected error
+            tags: ["Captcha"]
+        post:
+            operationId: validateSimpleCaptcha
```

Comment:
Following the existing POST API examples in Liferay, Seems like `postSimpleCaptcha()` is better?